Cargo.toml

```diff
@@ -51,6 +51,7 @@ futures = { version = "0.3.28", features = [
     "executor",
 ], default-features = true }
 garde = { version = "0.11.2", features = ["derive"], default-features = false }
+rand = "0.8.5"
```

Comment:
I think this is unused, and even if it was used, we use getrandom now.

Reply:
We use that to generate a random number betweem to numbers.

I think getrandom does not provide that, right?

Reply:
I see... Well, we can keep that, then!